source/github-helpers/get-pr-info.ts

```diff
@@ -2,7 +2,6 @@ import * as api from './api';
 import {getConversationNumber} from '.';
 
 export type PullRequestInfo = {
-	// TODO: Use this for `restore-file`
```

Comment:
When we added this line of code we needed to wait for something on GHE. @fregante are we good to drop this now?